README.rst

```diff
@@ -17,6 +17,14 @@ Then, to enable this plugin, run::
 
     tutor plugins enable notes
 
+Then, save config::
+
+    tutor config save
```

Comment:
There is no need to run `config save` because it is part of `local launch`.

Reply:
Thanks for the feedback removed unnecessary step.